source/lex.tex

```diff
@@ -968,7 +968,10 @@
 \indextext{\idxcode{_}|see{character, underscore}}%
 \indextext{character!underscore!in identifier}%
 \indextext{reserved identifier}%
-In addition, some identifiers are reserved for use by \Cpp{}
+In addition, some identifiers
+%FIXME: "appearing" isn't quite right here - reword?
+appearing as \grammarterm{token}{s} or \grammarterm{pp-tokens}
```

Comment:
I'm not sure I'm comfortable with this change from the approved wording. Thoughts? Maybe leave it to an issue?

Reply:
That said, the approved "pp-token" isn't actually an existing grammar term. @jensmaurer Are you OK with the change?

Reply:
We want to say that the (entire) token or pp-token is an identifier for the "reserved" rule to trigger. Except that pp-token (as in the issue write-up) doesn't exist; it's _preprocessing-token_. So, changing to `\grammarterm{preprocessing-token}{s} would be substantially better. (We mean the plural of _preprocessing-token_, not a sequence of them, which would be _pp-tokens_.

@tkoeppe, can you fix that, please?

Reply:
Yes, thanks! Please see what I merged, I think it both fixes this mistake and sticks as close to the approved wording as possible.